dev/breeze/pyproject.toml

```diff
@@ -51,6 +51,9 @@ dependencies = [
     "flit>=3.5.0",
     "gitpython>=3.1.40",
     "hatch==1.9.1",
+    # Importib_resources 6.2.0 break pytest_rewrite
+    # see https://github.com/python/importlib_resources/issues/299
+    "importlib_resources>=5.2,<6.2.0;python_version<\"3.9\"",
```

Comment:
```suggestion
    "importlib_resources>=5.2,!=6.2.0,!=6.3.0;python_version<\"3.9\"",
```

Reply:
imho these versions are broken (and will hopefully be fixed in a 6.3.1 release) since they introduced breaking changes without warning and in a minor release

Reply:
Maybe yes, maybe no. Maybe 6.3.1 will be fixed, and maybe 6.4.0 will be broken and only 6.5.0 will fix it. I craeted the issue and have not yet heard from importlib_resources maintainers. Or maybe pytest will release new version that is not crashing on 6.2.0 or 6.3.0 and we add `pytest >= x.y.z`.

This is pure speculation.

This is `living` main and we can always change it in the future when we know how to deal with it. For now limiting it to < 6.2 is safer and more accurate (for now we know that below 6.2.0 we are safe).